test/mountain_view/component_test.rb

```diff
@@ -17,15 +17,95 @@ def test_styleguide_stubs
     component = MountainView::Component.new("header")
 
     expected_stub = {
-      id: 1,
-      title: "20 Mountains you didn't know they even existed",
-      subtitle: "Buzzfeed title"
-    }
+                      :meta=>"There is this different classes",
+                      :stubs=>
+                        [
+                          {:id=>1, :title=>"20 Mountains you didn't know they even existed",
+                           :subtitle=>"Buzzfeed title"},
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.<br>Space inside } missing.